object_manager.go

```diff
@@ -129,6 +130,33 @@ func (objMgr *ObjectManager) GetNetworkView(name string) (*NetworkView, error) {
 	return &res[0], nil
 }
 
+func (objMgr *ObjectManager) UpdateNetworkView(ref string, addEA EA, removeEA EA) error {
```

Comment:
1. Ok I will rename it to UpdateNetworkViewEA . Right now I dont have any need to update the netview view.

2. Yes I did not add it because I did not find it generic

3. Makes sense. I will create a generic method to update EA for the IBObjects and then i will not need a specific method for updating NetworkView EAs. Thanks!